RecoTracker/LSTCore/src/LSTESData.cc

```diff
@@ -111,11 +113,31 @@ std::unique_ptr<lst::LSTESData<alpaka_common::DevHost>> lst::loadAndFillESHost(s
                                                  tiltedGeometry,
                                                  moduleConnectionMap);
   auto pixelMappingPtr = std::make_shared<PixelMap>(std::move(pixelMapping));
+
+  // === Load from the DNN instance ===
+  auto model = Dnn<DenseLayer<23, 32>, DenseLayer<32, 32>, DenseLayer<32, 1>>(
+      "/mnt/data1/gsn27/cmssw-fresh/CMSSW_14_2_0_pre4/src/RecoTracker/LSTCore/standalone/analysis/DNN/"
+      "network_weights.bin");
+
+  // Copy the loaded model into a host DnnWeightsDevData struct
+  lst::DnnWeightsDevData hostDnn;
+  {
+    auto const& layers = model.getLayers();
+    hostDnn.layer1 = std::get<0>(layers);
+    hostDnn.layer2 = std::get<1>(layers);
+    hostDnn.layer3 = std::get<2>(layers);
+  }
```

Comment:
I'm curious if there is a way to define the parameter pack and index relationship in `DnnWeightsDevData` to simplify this code